backend/common/src/main/java/ai/verta/modeldb/common/config/RdbConfig.java

```diff
@@ -109,7 +109,7 @@ public static String buildDatabaseServerConnectionString(RdbConfig rdb) {
     }
 
     if (rdb.isH2()) {
-      return rdb.RdbUrl + ";DB_CLOSE_DELAY=-1";
+      return rdb.RdbUrl + ";DB_CLOSE_DELAY=-1;CASE_INSENSITIVE_IDENTIFIERS=TRUE";
```

Comment:
is this the change that means we don't to uppercase the tables any more?

Reply:
Yes. Correct